### DISCUSSION TITLE
ts 校验不通过：Property 'children' does not exist on type 'Readonly<{}>'.ts(2339)

### DISCUSSION THREAD

**Discussion by wenxiuYang** [NervJS/taro#12341] [Q & A]
### 相关平台
H5

### 复现仓库
[https://github.com/wenxiuYang/taro-demo/blob/movablearea-demo/src/app.ts](https://github.com/wenxiuYang/taro-demo/blob/movablearea-demo/src/app.ts)
**浏览器版本: 104.0.5112.102（正式版本） （64 位）**
**使用框架: React**

### 复现步骤
使用 【this.props.children】时 ts 出现警告：Property 'children' does not exist on type 'Readonly<{}>'.ts(2339)

### 期望结果
使用【this.props.children】时，ts 可以校验通过

### 实际结果
使用【this.props.children】时，ts 校验不通过

### 环境信息
```
Taro CLI 3.5.3 environment info:
    System:
      OS: Windows 10
    Binaries:
      Node: 14.17.4 - 
      Yarn: 1.22.19 - 
      npm: 6.14.14 - 
```

**Comment by cookie-zhang** [comment]
1.  "@types/react": "^18.0.15", 升级一下这个试
 2. 或者定义一个interface Iprops {cheildren?: any}

**Reply by wenxiuYang** [reply]
方法2 是可以解决这个问题的

**Comment by ZakaryCode** [comment]
所示示例与问题不符，且无法复现该问题，可尝试升级或参考上一开发者回复修复问题

**Reply by wenxiuYang** [reply]
已经是最新的taro版本（V 3.5.3）了哦，楼上说的手动添加interface 可以解决此问题
但是我这demo 是 `taro init projectName`创建的，后续项目总不能一直手动添加interface 吧
还是麻烦解决一下呢
